include/z64actor.h

```diff
@@ -1002,6 +1007,69 @@ typedef enum {
     /* 0x2B2 */ ACTOR_ID_MAX // originally "ACTOR_DLF_MAX"
 } ActorID;
 
+// Targetability / ACTOR_FLAG_TARGETABLE?
+#define ACTOR_FLAG_1             (1 << 0)
+// 
+#define ACTOR_FLAG_2             (1 << 1)
+// 
+#define ACTOR_FLAG_4             (1 << 2)
+// 
+#define ACTOR_FLAG_8             (1 << 3)
+// 
+#define ACTOR_FLAG_10            (1 << 4)
+// 
+#define ACTOR_FLAG_20            (1 << 5)
+// 
+#define ACTOR_FLAG_40            (1 << 6)
+// Invisible
+#define ACTOR_FLAG_80            (1 << 7)
+// Related to talk
```

Comment:
In OoT, these were recently brought into the repo as `ACTOR_FLAG_X, with X = 1, 2, 3, 4, 5, 6, 7, 8... instead of X = 1, 2, 4, 8, 10, 20, 40, 80, ...

I do think the former is nicer in the finished repo as the names are shorter, but the latter one is nicer in a WIP repo as the names are easier to map to the flags. But doing that loses consistency. Did you have thoughts on this?

Reply:
Yeah, I noted OoT does something different here, and I think we shouldn't do the same for a few reasons

- There's no guarantee 2 flags with the same number represent the same flag in both games (0x1000 in OoT could mean something different in MM).
  - This way, if anybody tried to copy an actor from one game to the other, it would trigger a compilation error instead of silently triggering wrong behaviour
- As you said, OoT has finished reached 100% while we are very far behind, and I think having those flags as the hex number they correspond is easier to use when decomping
- The names aren't supposed to be final. Hopefully those will be named soon with more OoT and MM passes.

I'm open to suggestions tho, would be nice to hear more opinions on this topic.

Reply:
At the moment, the systems are to dissimilar for people to accidentally break the game importing actors, so I don't feel like we should be worried too much about what people are doing modding wise.

I have an issue with how the names don't make it clear that they represent a hexadecimal mask instead of a "left shift" like OoT does. When representing bit flags in hexadecimal in my own personal notes, my style is to add leading zeros to align with the most significant byte, as octal is never used and it makes it more immediately clear that the number represents a hexadecimal mask.

Reply:
I recall us having a discussion about whether we wanted to have bitwise flags include leading zeros or not as you prefer. There was support for both sides of it, but not having leading zeros edged out a majority preference. Of course, this could always change as we get more contributors.

I prefer not having leading zeros because to me, the "FLAG" is enough to indicate that it's bitwise as more often than not, that's what flags are doing in decomp. And past that, extra zeros just add space. Plus, it's easy to look at defines for these as well if there's ever uncertainty. 

With that, I think I'm fine leaving them as is for now. Others are welcome to share their preferences as well